### PR TITLE
Reviews comments

### PR DESCRIPTION
**Pull request by RamiroPinol**
Feature: write comments when reviews are approved or rejected [175563048](https://www.pivotaltracker.com/n/projects/2463484/stories/175563048)

### CODE REVIEW

**Comment by mattvv** [comment]
Direction looking great :)

**Comment by RamiroPinol** [comment]
@mattvv I think this is ready. Code flow ended up a bit messy IMO, any feedback is welcomed :)

File: components/Reviews.tsx [hunk 1]
```diff
@@ -63,30 +72,94 @@ const Reviews = ({ reviews, storyId, currentState, updateStory }: ReviewParams):
...
+    if (originalReview && originalReview.status === updatedReview.status) {
+      return;
+    }
+
+    // Pivotal default comment for Pass or Revise status
+    let comment = `**${reviewTypes[updatedReview.review_type_id].name}** review set to **${
+      updatedReview.status
+    }**`;
+
+    // If user writed a comment, we add it
```
Comment (mattvv) [inline components/Reviews.tsx:101]:
```suggestion
// If user wrote a comment, we add it
```

**Review by mattvv — APPROVED** [review]
:shipit:

File: components/Reviews.tsx (hunk 1 above)
Comment (RamiroPinol) [reply components/Reviews.tsx:101]: 🤦🏻

**Pull request merged**